pkg/services/provisioning/dashboards/file_reader.go

```diff
@@ -333,12 +333,12 @@ func (fr *fileReader) resolvePath(path string) string {
 	copy := path
 	path, err := filepath.Abs(path)
 	if err != nil {
-		fr.log.Error("Could not create absolute path ", "path", path)
+		fr.log.Error("Could not create absolute path", "path", copy, "error", err)
```

Comment:
Note that when error happens, the result `path` will likely be empty, so print `copy` instead.